config/gitter-config-sample.yaml

```diff
@@ -28,3 +28,7 @@ logging:
     files:
         -   "debug.log": "debug"
     maxFiles: 1
+
+
+enable_portals: true
```

Comment:
Should the sample contain `false` here instead so that someone who just copies the sample needs to make a conscious decision to enable portals?

Reply:
Eh, I guess so.